build_web_compilers/lib/src/dart2js_bootstrap.dart

```diff
@@ -162,3 +165,37 @@ Future<void> _copyIfExists(
 }
 
 final _resourcePool = Pool(maxWorkersPerTask);
+
+const _dart2jsVmArgsEnvVar = 'BUILD_DART2JS_VM_ARGS';
```

Comment:
Can you call this out in the commit message? Why do we need this?

Reply:
Oh yes this was old behavior I am also restoring, workiva I believe needed it to increase ram available to dart2js.